docs/content/how-to/upgrades.md

```diff
@@ -1,27 +1,34 @@
 
-For a cluster to keep fully operational during and upgrade process, Control Plane and Nodes upgrades need to be orchestrated while satisfying [Kubernetes version skew policy](https://kubernetes.io/releases/version-skew-policy/) at any time.
+For a cluster to keep fully operational during an upgrade process, Control Plane and Nodes upgrades need to be orchestrated while satisfying [Kubernetes version skew policy](https://kubernetes.io/releases/version-skew-policy/) at any time. The supported OCP versions are dictated by the running HyperShift Operator [see here](../reference/versioning-support.md) for more details on versioning.
 
 ## HostedCluster
-
 `.spec.release` dictates the version of the Control Plane.
 
 The HostedCluster propagates the intended `.spec.release` to the `HostedControlPlane.spec.release` and runs the appropriate Control Plane Operator version.
 
-The HostedControlPlane orchestrates the rollout of the new version of the Control Plane components along with any OCP component through the new version of the [CVO](https://github.com/openshift/cluster-version-operator).
+The HostedControlPlane orchestrates the rollout of the new versions of the Control Plane components along with any OCP component through the new version of the [cluster version operator (CVO)](https://github.com/openshift/cluster-version-operator). This includes resources like:
 
-## NodePools
+- the CVO itself
+- cluster network operator (CNO)
+- cluster ingress operator
+- manifests for the kube API-server (KAS), scheduler, and manager
+- machine approver
+- autoscaler
+- infra resources needed to enable ingress for control plane endpoints (KAS, ignition, konnectivity, etc.)
 
-`.spec.release` dictates the version of any particular NodePool.
+The CVO also applies the payload for the `CLUSTER_PROFILE": "ibm-cloud-managed` into the guest cluster.
 
-A NodePool will perform a Replace/InPlace rolling upgrade according to `.spec.management.upgradeType`.
+Traditionally, in standalone OCP, the CVO has been the sole source of truth for upgrades. In HyperShift, the responsibility is currently split between CPO and CVO. This enabled the flexibility and speed the HyperShift project needed for the CPO to support the management/guest cluster topology and the multiple customizations needed for manifest that otherwise would have needed to be segregated in the payload.
+
+HyperShift bubbles up the HC.Status info coming from CVO/cincinnati for available upgrades. This info is purely informational and doesn’t determine upgradability input, which is dictated by the `.spec.release` input in practice. This does result in the loss of some of the builtin features and guardrails from CVO like recommendations, allowed upgrade paths, risks, etc. However, this information is still available to read for consumers in this HC.Status field.
+
+## NodePools
```

Comment:
Do we have any upgrade restrictions on NodePools with respect to the HostedCluster release version?

Reply:
Yes, same as kubelet skew: https://kubernetes.io/releases/version-skew-policy/#kubelet

Reply:
Thanks! I missed where we mentioned that above 🤦🏻